static/js/components/SubscriptionsList.js

```diff
@@ -45,7 +50,11 @@ export default class SubscriptionsList extends React.Component<Props> {
       <div className="location-list subscribed-channels">
         {myChannels.length > 0 ? (
           <div className="my-channels">
-            <div className="heading">Channels you moderate</div>
+            <NavigationItem
+              whenExpanded={() => (
+                <div className="heading">Channels you moderate</div>
+              )}
+            />
```

Comment:
Can you do this also for the `Channels` header below, to avoid this:

<img width="293" alt="Screen Shot 2019-04-26 at 11 23 19 AM" src="https://user-images.githubusercontent.com/187676/56818584-ec536100-6815-11e9-9664-e706389a5d52.png">

Reply:
oh haha, I missed that b/c on my normal dev user i'm not subscribed to any channels I'm not moderating :woman_facepalming: